src/main/charts/bitbucket/values.yaml

```diff
@@ -52,6 +52,12 @@ bitbucket:
     port: 80
     # -- The type of Kubernetes service to use for Bitbucket
     type: ClusterIP
+    annotations: {}
+  sshService:
```

Comment:
I think we definitely need some inline documentation in the values file so users can understand how and when these values should be utilised.

Beyond the inline comments, I think we should have a separate section in configuration explaining SSH setup in kubernetes as it is a bit mindbending experience :)

Reply:
@nanux do you mean port forwarding? In my clusters I follow https://kubernetes.github.io/ingress-nginx/user-guide/exposing-tcp-udp-services/ but instructions will vary depending on the ingress itself.

Reply:
Added some inline documentation for these properties